source/content/guides/drush/03-drush-commands.md

```diff
@@ -106,4 +106,4 @@ You should place Drush commands in `drush/Commands` if using Drupal 9.
 ## More Resources
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drush/drush-commands